docs/production-deployment/cloud/get-started/namespaces.mdx

```diff
@@ -251,14 +251,29 @@ Sample workflowId convention:
 Temporal Cloud normally supports authentication to Namespaces using [API keys](/cloud/api-keys) _or_ [mTLS](/cloud/certificates).
 If you need to migrate from one authentication method to another, or you require both API key and mTLS authentication to be enabled on your Namespace, please contact [Support](https://docs.temporal.io/cloud/support#support-ticket).
 
+:::info
+
+Requiring both API key and mTLS is in [public preview](/evaluate/development-production-features/release-stages), and doesn't currently support [High Availability features](/cloud/high-availability).
+
+:::
+
 See the documentation for [API keys](/cloud/api-keys) and [mTLS certificates](/cloud/certificates) for more information on how to create and manage your credentials.
 
 Authentication methods require specific endpoints in order to programmatically access your Namespace.
+There are two types of gRPC endpoints for accessing a Namespace in Temporal Cloud: a namespace endpoint (`<namespace>.<account>.tmprl.cloud:7233`) and a regional endpoint (`<region>.<cloud_provider>.api.temporal.io:7233`).
+Which one to use depends on your authentication method and whether your Namespace has [High Availability features](/cloud/high-availability) enabled, as shown in the table below.
 
-- For the API key authentication method, use the gRPC regional endpoint `<region>.<cloud_provider>.api.temporal.io:7233`.
-- For the mTLS authentication method, use the gRPC Namespace endpoint `<namespace>.<account>.tmprl.cloud:7233`.
-- For [Namespaces with High Availability features](/cloud/high-availability) use the gRPC Namespace endpoint `<namespace>.<account>.tmprl.cloud:7233`, regardless of your authentication method.
-  This allows automated failover without needing to switch your endpoint.
+|                        | Not High Availability | High Availability                                                                |
+|------------------------|-----------------------|----------------------------------------------------------------------------------|
+| mTLS Authentication    | Namespace             | Namespace                                                                        |
+| API Key Authentication | Regional              | Both work, but Namespace is recommended because it allows for automated failover |
```

Comment:
```suggestion
| API Key Authentication | Regional             | Both work, but Namespace is recommended because it reduces the unavailability window during a failover event |
```